src/atoms/Button.js

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+
+function Button() {
+    return (
+        <button className="button">Button Lor </button>
```

Comment:
This always renders a button with the text "Button Lor". Instead, allow to use any text:

```jsx
function Button(props) {
  return <button className="button">{props.children}</button>
}
```

Now you can render your `Button` component with different text anywhere:

```jsx
<Button>Foo</Button>
<Button>This is another text</Button>
```